<?xml version="1.0"?>
<launch>
  <arg name="config_file_path" default="$(find metrics_refbox)/config/" />
  <arg name="results_file_path" default="$(find metrics_refbox)/results/" />
  <arg name="config_file" default="refbox.json" />

  <node pkg="metrics_refbox" type="metrics_refbox_node"
        name="metrics_refbox" output="screen">
    <param name="results_file_path" type="str" value="$(arg results_file_path)"/>
    <param name="config_file_path" type="str" value="$(arg config_file_path)"/>
    <param name="config_file" type="str" value="$(arg config_file)"/>
    <param name="loop_rate" type="double" value="1.0"/>
  </node>
</launch>
